### ISSUE TITLE
Duplicated origin for ... when building with Python 3

### ISSUE THREAD

**Issue by rumpelsepp** [freebsd/poudriere#402]
Hey,

I have configured my own, awesome poudriere server. I have this `make.conf`:

```
DEFAULT_VERSIONS+= ssl=libressl
DEFAULT_VERSIONS+= python=3.5
WITHOUT_MODULES = sound ntfs linux
```

When I try to build all packages, with `poudriere bulk -j freebsd_10-3x64 -p HEAD -a` it fails and I get the following errors:

```
[00:00:48] ====>> Error: Duplicated origin for py35-libarchive-c-2.2: archivers/py3-libarchive-c AND archivers/py-libarchive-c. Rerun with -vv to see which ports are depending on these.
[00:04:04] ====>> Error: Duplicated origin for py35-bsddb3-6.2.1: databases/py3-bsddb3 AND databases/py-bsddb3. Rerun with -vv to see which ports are depending on these.
[00:04:05] ====>> Error: Duplicated origin for py35-gdbm-3.5.2_4: databases/py35-gdbm AND databases/py-gdbm. Rerun with -vv to see which ports are depending on these.
[00:04:05] ====>> Error: Duplicated origin for py35-sqlite3-3.5.2_7: databases/py35-sqlite3 AND databases/py-sqlite3. Rerun with -vv to see which ports are depending on these.
[00:09:02] ====>> Error: Duplicated origin for py35-Jinja2-2.8: devel/py3-Jinja2 AND devel/py-Jinja2. Rerun with -vv to see which ports are depending on these.
[00:09:02] ====>> Error: Duplicated origin for py35-Babel-2.3.4: devel/py3-babel AND devel/py-babel. Rerun with -vv to see which ports are depending on these.
[00:09:02] ====>> Error: Duplicated origin for py35-jsonschema-2.5.1: devel/py3-jsonschema AND devel/py-jsonschema. Rerun with -vv to see which ports are depending on these.
[00:09:02] ====>> Error: Duplicated origin for py35-nose-1.3.7: devel/py3-nose AND devel/py-nose. Rerun with -vv to see which ports are depending on these.
[00:09:02] ====>> Error: Duplicated origin for py35-python-magic-0.4.10: devel/py3-python-magic AND devel/py-python-magic. Rerun with -vv to see which ports are depending on these.
[00:09:02] ====>> Error: Duplicated origin for py35-pyicu-1.8_3: devel/py3-pyicu AND devel/py-pyicu. Rerun with -vv to see which ports are depending on these.
[00:09:02] ====>> Error: Duplicated origin for py35-pytz-2016.4,1: devel/py3-pytz AND devel/py-pytz. Rerun with -vv to see which ports are depending on these.
[00:09:50] ====>> Error: Duplicated origin for py35-sqlite3-3.5.2_7: databases/py35-sqlite3 AND databases/py-sqlite3. Rerun with -vv to see which ports are depending on these.
[00:13:32] ====>> Error: Duplicated origin for py35-pygraphviz-1.3.1: graphics/py3-pygraphviz AND graphics/py-pygraphviz. Rerun with -vv to see which ports are depending on these.
[00:13:32] ====>> Error: Duplicated origin for py35-pillow-3.1.1_1: graphics/py3-pillow AND graphics/py-pillow. Rerun with -vv to see which ports are depending on these.
[00:18:31] ====>> Error: Duplicated origin for py35-netifaces-0.10.4: net/py3-netifaces AND net/py-netifaces. Rerun with -vv to see which ports are depending on these.
[00:20:26] ====>> Error: Duplicated origin for py35-bsddb3-6.2.1: databases/py3-bsddb3 AND databases/py-bsddb3. Rerun with -vv to see which ports are depending on these.
[00:22:23] ====>> Error: Duplicated origin for py35-libarchive-c-2.2: archivers/py3-libarchive-c AND archivers/py-libarchive-c. Rerun with -vv to see which ports are depending on these.
[00:23:47] ====>> Error: Duplicated origin for py35-MarkupSafe-0.23: textproc/py3-MarkupSafe AND textproc/py-MarkupSafe. Rerun with -vv to see which ports are depending on these.
[00:27:25] ====>> Error: Duplicated origin for py35-tkinter-3.5.2_6: x11-toolkits/py35-tkinter AND x11-toolkits/py-tkinter. Rerun with -vv to see which ports are depending on these.
```

What exactly is the problem, and how can I fix this? Thanks!

**Comment by bdrewery** [comment]
This is a consequence of how these ports were added. It's a ports bug. One of them needs an IGNORE when the default is already 3. Let freebsd-python@FreeBSD.org or #freebsd-python on Freenode know.

**Comment by bdrewery** [comment]
Fixed by #483
